tests/zipkin_test.py

```diff
@@ -844,6 +844,48 @@ def test_override_span_name(self):
             assert span.span_name == 'new_name'
             assert span.logging_context.span_name == 'new_name'
 
+    def test_create_http_headers_for_my_span_no_zipkin_attrs(self):
+        with zipkin.zipkin_client_span(
+                service_name='test_service',
```

Comment:
too much indentation. Please rebase on the black branch and run `tox -e black` to reformat this code